NGitLab.Mock/Clients/GitLabClient.cs

```diff
@@ -71,6 +71,8 @@ public IGraphQLClient GraphQL
 
         public IMilestoneClient GetMilestone(int projectId) => new MilestoneClient(Context, projectId);
 
+        public IMilestoneClient GetGroupMilestone(int groupId) => new MilestoneClient(Context, groupId);
```

Comment:
This calls the same ctor as GetMilestone. We need to distinguish between group and project scope.

Reply:
for the test, i'll need to talk to you.